src/MOI_wrapper/MOI_wrapper.jl

```diff
@@ -1841,9 +1858,9 @@ function MOI.delete(
     ret =
         GRBsetcharattrelement(model, "VType", Cint(info.column - 1), Char('C'))
     _check_ret(model, ret)
-    _require_update(model)
     info.type = GRB_CONTINUOUS
     model.name_to_constraint_index = nothing
+    _update_if_necessary(model, force = true)
```

Comment:
Are these `force` necessary if we're just changing a variable from binary to continuous?